suricata.yaml.in

```diff
@@ -281,6 +281,16 @@ outputs:
             # DNS record types to log, based on the query type.
             # Default: all.
             #types: [a, aaaa, cname, mx, ns, ptr, txt]
+        - mdns:
+            #requests: true
+            #responses: true
+            #grouped: false
+            #additionals: false
+            #authorities: false
+            # Don't log queries in responses.
+            #queries-in-response: false
+            # Don't log answers in requests
+            #answers-in-request: false
```

Comment:
I'm a bit hesitant to expose too many parameters here?  Maybe just a `verbose` flag to turn everything on?  If jumping into to get a picture of a network quickly, stuff like answers-in-request, could be very useful.  But just added noise when running for a long time.

So thoughts on breaking out the flags this way? Or just `verbose: false`.